config/features.rb

```diff
@@ -7,46 +7,6 @@
     default: false,
```

Comment:
is debug still used anywhere? I think I saw the partial pulled out but I suppose it may be used elsewhere...